### DISCUSSION TITLE
Allow editing commit messages after merge squash

### DISCUSSION THREAD

**Discussion by mrswats** [pwntester/octo.nvim#264] [Ideas]
In the GitHub UI you are allowed to modify the commit message before you merge squash a pull request. This capability was added in the CLI: https://github.com/cli/cli/issues/1023, but I couldn't find the option to do this in Octo. Did I miss it? It is not implemented?

Thanks for the help in advance. Absolutely love the plugin.

**Comment by pwntester** [comment]
Yep, that feature is not implemented yet. Please file an issue and will try to implement it soon

**Reply by mrswats** [reply]
Awesome, will do. Thank you :)
